sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb

```diff
@@ -0,0 +1,1337 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "ml_client.jobs.download(job_name, download_path=\".\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "fcst_df = pd.read_json(output_file, orient=\"table\")\n",
+        "fcst_df.head()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Calculate the metrics."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from metrics_helper import calculate_metrics\n",
+        "\n",
+        "calculate_metrics(fcst_df[target_column_name], fcst_df[\"predicted\"])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Forecast versus actuals plot.\n",
+        "We will join historical data with the predictions to plot predictions and actuals on a time series plot."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "history_data = pd.read_csv(\n",
+        "    \"./data/training-mltable-folder/dominicks_OJ_train.csv\",\n",
+        "    parse_dates=[time_column_name],\n",
+        ")\n",
+        "history_data.sort_values(by=time_column_name, inplace=True)\n",
+        "history_data = history_data.iloc[-3 * forecast_horizon :]\n",
+        "# Merge predictions to historic data.\n",
+        "df = pd.concat([history_data, fcst_df], sort=False, ignore_index=True)\n",
+        "df.set_index(time_column_name, inplace=True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Build the plot"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "%matplotlib inline\n",
+        "from matplotlib import pyplot as plt\n",
+        "\n",
+        "plt.plot(df[[target_column_name, \"predicted\"]])\n",
+        "plt.xticks(rotation=45)\n",
+        "plt.title(f\"Predicted vs. Actuals\")\n",
+        "plt.legend([\"actual\", \"forecast\"])\n",
+        "plt.show()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Delete the batch endpoint and compute. Do not do it occasionally.\n",
+        "ml_client.batch_endpoints.begin_delete(name=batch_endpoint_name).wait()\n",
+        "ml_client.compute.begin_delete(name=cluster_name).wait()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 6.3 Deployment\n",
```

Comment:
Please replace the entire Section 6.3 with the contents of [bike-share notebook](https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb)